### DISCUSSION TITLE
File name too long in tmp file upload

### DISCUSSION THREAD

**Discussion by kannerban** [livewire/livewire#2238] [1. Help]
i have problem with tmp file name in other language. cause this 

message | "fopen(/var/www/html/multiup/storage/app/livewire-tmp/LYCKtev7fgBxuTDjG2EJcNsSbs6bKD-metaMjAyMC0xMi0wOS3guKPguLLguKLguIfguLLguJnguKrguKPguLjguJvguJzguKXguIHguLLguKPguJTguLPguYDguJnguLTguJnguIHguLLguKPguIjguLHguJTguIvguLfguYnguK3guIjguLHguJTguIjguYnguLLguIfguYPguJnguKPguK3guJrguYDguJTguLfguK3guJkg4Lie4Lik4Lio4LiI4Li04LiB4Liy4Lii4LiZ4LieLuC4qC4yNTYzLnBkZg==-.pdf):  failed to open stream: File name too long"
-- | --

where can i static tmp file name or convert to utf8?

**Comment by oxytoxin** [comment]
I am having the same issue, any way to fix this?

**Comment by ZerglingGo** [comment]
I created an issue on this problem: #2834
